test/helpers.lua

```diff
@@ -290,22 +286,28 @@ module.uname = (function()
     if status then
       platform = string.lower(f:read("*l"))
       f:close()
-    elseif module.iswin() then
+    elseif package.config:sub(1,1) == '\\' then
       platform = 'windows'
     else
       error('unknown platform')
     end
     return platform
-  end)
-end)()
+  end
+end
 
 function module.is_os(s)
-  if not (s == 'win' or s == 'mac' or s == 'unix') then
+  if not (s == 'win'
+    or s == 'mac'
+    or s == 'freebsd'
+    or s == 'openbsd'
+    or s == 'bsd') then
     error('unknown platform: '..tostring(s))
   end
-  return ((s == 'win' and module.iswin())
+  return ((s == 'win' and module.uname() == 'windows')
     or (s == 'mac' and module.uname() == 'darwin')
-    or (s == 'unix'))
```

Comment:
The reason for this was because `is_os('unix')` is usually expected to return true for non-windows. Do we still have that behavior?